backend/btxn-service-rest/src/main/webapp/WEB-INF/web.xml

```diff
@@ -21,4 +21,24 @@
       xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
       version="3.0">
 
+  <!-- Security -->
+  <security-constraint>
+    <web-resource-collection>
+      <web-resource-name>hawkular-btm-btxn-service-rest</web-resource-name>
+      <url-pattern>/*</url-pattern>
+    </web-resource-collection>
+    <auth-constraint>
+      <role-name>user</role-name>
+    </auth-constraint>
+  </security-constraint>
+
+  <login-config>
+    <auth-method>BASIC</auth-method>

```

Comment:
This should be "KEYCLOAK". I wonder how it worked for you with this, as I remember a code on Keycloak's side that checked for this value. See http://www.hawkular.org/docs/dev/accounts.html